### PR TITLE
Add pagedreport to README

### PR DESCRIPTION
**Pull request by tvroylandt**
This PR add a line to the README about some potential use of `pagedown` : https://github.com/rfortherestofus/pagedreport

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,8 @@ Want more poster styles? You may take a look at [Brent Thorne's **posterdown** p
 
 - Beatriz Milz's CV: https://beatrizmilz.github.io/resume/resume.html
 
+- `pagedreport` by Thomas Vroylandt and David Keyes, which help using `pagedown` with some templates proposals : https://github.com/rfortherestofus/pagedreport
```
Comment (RLesur) [inline README.md:74]:
```suggestion
- **pagedreport** by Thomas Vroylandt and David Keyes which helps using **pagedown** with some templates proposals: https://github.com/rfortherestofus/pagedreport
```

**Review by RLesur — CHANGES_REQUESTED** [review]

**Review by RLesur — APPROVED** [review]

**Comment by cderv** [comment]
Awesome! Thanks @tvroylandt ! 
Congrats on the release!

**Comment by RLesur** [comment]
Thanks @tvroylandt! Beautiful job!

**Comment by tvroylandt** [comment]
Thank you !

**Pull request merged**